doc/source/rllib/getting-started.rst

```diff
@@ -0,0 +1,449 @@
+the :ref:`Ray Tune API <tune-api-ref>`.
+
+This allows for easy utilization of your configured :py:class:`~ray.rllib.algorithms.algorithm.Algorithm` in
+:ref:`Ray Tune <tune-main>` experiments.
+
+For example, the following code performs a simple hyper-parameter sweep of your :ref:`PPO <ppo>`
+through creating three ``Trials``, one for each configured learning rate:
+
+.. testcode::
+
+    from ray import train, tune
+    from ray.rllib.algorithms.ppo import PPOConfig
+
+    config = (
+        PPOConfig()
+        .environment("Pendulum-v1")
+        # Specify a simple tune hyperparameter sweep.
+        .training(
+            lr=tune.grid_search([0.001, 0.0005, 0.0001]),
+        )
+    )
+
+    # Create a Tuner instance to manage the trials.
+    tuner = tune.Tuner(
+        config.algo_class,
+        param_space=config,
+        # Specify a stopping criterion. Note that the criterion has to match one of the
+        # pretty printed result metrics from the results returned previously by
+        # ``.train()``.
+        run_config=train.RunConfig(
+            stop={"env_runners/episode_return_mean": -1000.0},
+        ),
+    )
+    # Run the Tuner and capture the results.
+    results = tuner.fit()
+
+Note that each :py:class:`~ray.tune.trial.Trial` creates a separate
+:py:class:`~ray.rllib.algorithms.algorithm.Algorithm` instance as a :ref:`Ray actor <actor-guide>`,
+assigns compute resources to each ``Trial``, and runs them in parallel, if possible,
+on your Ray cluster:
+
+.. code-block:: text
+
+    Trial status: 3 RUNNING
+    Current time: 2025-01-17 18:47:33. Total running time: 3min 0s
+    Logical resource usage: 9.0/12 CPUs, 0/0 GPUs
+    ╭──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
+    │ Trial name                    status         lr     iter     total time (s)  episode_return_mean    ..._sampled_lifetime │
+    ├──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┤
+    │ PPO_Pendulum-v1_b5c41_00000   RUNNING    0.01         29            86.2426             -998.449                  108000 │
+    │ PPO_Pendulum-v1_b5c41_00001   RUNNING    0.001        25            74.4335             -997.079                  100000 │
+    │ PPO_Pendulum-v1_b5c41_00002   RUNNING    0.0001       20            60.0421             -960.293                   80000 │
+    ╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+
+``Tuner.fit()`` returns an ``ResultGrid`` object that allows for a detailed analysis of the
+training process and for retrieving the :ref:`checkpoints <rllib-checkpoints-docs>` of the trained
+algorithms and their models:
+
+.. testcode::
+    # Get the best result based on a particular metric.
+    best_result = results.get_best_result(
```

Comment:
Maybe we set `scope="last"` in `get_best_result` explicitly to clarify these are the last iterations from each trial we choose the best result from?

Reply:
Ok, good idea.

Reply:
done